### PR TITLE
Sim spark max limits

### PR DESCRIPTION
**Pull request by LucienMorey**

### CODE REVIEW

**Comment by LucienMorey** [comment]
This does the limits thing properly but the sim doesn't seem to be capable of hitting the setpoint to trigger the forward limit on the extension or the required min angle to transition in the scoring state machine. Ideas on this would be appreciated

**Review by auscompgeek — APPROVED** [review]
Oh good I see the FaultID enum is in the bindings 😄

code LGTM, I'll leave it to Oliver to work out why it wouldn't be hitting the end limit

**Comment by LucienMorey** [comment]
I adjusted the mass of the arm to be 2.5 kgs which is basically what it is according to the cad...
image

**Comment by LucienMorey** [comment]
I made sure to account for the arm and now with the proper masses It seems to be capable of hitting both limits

File: physics.py [hunk 1]
```diff
@@ -92,10 +98,12 @@ def __init__(self, physics_controller: PhysicsInterface, robot: MyRobot):
             measurementStdDevs=[math.radians(0.01)],
         )
         extension_motors_sim = DCMotor.NEO550(1)
+        # This value was pulled from the CAD Model on 19/02/23
+        ELEVATOR_CARRIAGE_MASS = 0.655
         self.extension_sim = ElevatorSim(
             extension_motors_sim,
             robot.arm.EXTEND_GEAR_RATIO,
-            4,
+            ELEVATOR_CARRIAGE_MASS + END_EFFECTOR_MASS,
```
Comment (auscompgeek) [inline physics.py:106]: lol that was one hell of an estimate back when we wrote this :joy:

**Pull request merged**